projects/train/config.yaml

```diff
@@ -39,7 +39,7 @@ data:
 
     # preprocessing args
     batch_size: 384
-    batches_per_epoch: 200
+    batches_per_epoch: 800
```

Comment:
Do we want this higher to match (roughly) what we were doing before? And could you add this change and the `right_pad` change to the BBH/BNS training configs?